### ISSUE TITLE
[Linux] Forcing resolutions [TFC?]

### ISSUE THREAD

**Issue by kriswema** [ValveSoftware/halflife#1683]
So. 
I'm trying to start TFC with `-window -width 1920 -height 1080` and also tried adding `-stretchaspect`. 
I found [this wiki](https://developer.valvesoftware.com/wiki/Command_Line_Options#Goldsource_Games) that lists all start parameters, which is also where I found `-strechaspect`. I searched for an issue related to this and stumbled upon a post that said I needed to have the desired resolution registered in my operating system? That shouldn't be necessary for windowed mode, right? I expected to be able to set almost arbitrary resolutions with the parameters. And 1080p isn't really all that arbitrary.

TL;DR: TFC thinks it's ok to open as 1920x1381, even though I explicitly asked it not to. How rude.
